### PR TITLE
Dashboard: retrieve config using UUIDs

### PR DESCRIPTION
**Pull request by sevein**
This commit updates our `ingest.js` and `jobs.js` files so they don't
use labels for conditional behaviour. Instead, we're using UUIDs which
are going to be the same regardless the language used in the user
interface.

Also, this commit is removing some dead code:
- Review button in Store DIP (broken and unused).
- Dialog in unexistent micro-service:
  "Normalize submission documentation to preservation format".

This is connected to https://github.com/artefactual/archivematica/issues/1101, https://github.com/archivematica/Issues/issues/62.

### CODE REVIEW

**Comment by sevein** [comment]
For QA, after merging, it should be possible to verify that:

- "Approve normalization" includes Review button (only while waiting for approval).
- "Approve normalization" includes Report button.
- Clicking on "Upload DIP to Archivists Toolkit" opens new page as usual.
- Clicking on "Upload DIP to ArchivesSpace" opens new page as usual.
- Clicking on "Upload DIP to AtoM/Binder" opens new modal as usual.
- "Move to the uploadedDIPs directory" includes Review button.
- "Store AIP" includes Review button (while waiting for approval).

File: src/dashboard/src/media/js/ingest.js
Comment (sevein) [inline src/dashboard/src/media/js/ingest.js:188]: Unused.

File: src/dashboard/src/media/js/jobs.js
Comment (sevein) [inline src/dashboard/src/media/js/jobs.js]: `Store DIP` not included in my change because is unused - see https://github.com/archivematica/Issues/issues/62.

File: src/dashboard/src/components/transfer/views.py
Comment (ross-spencer) [inline src/dashboard/src/components/transfer/views.py:159]: A question outside of CR, but with these values in `transfer/views/py` and `ingest/views.py` could they be inherited from elsewhere instead of defining them twice?

File: src/dashboard/src/media/js/ingest.js
Comment (ross-spencer) [inline src/dashboard/src/media/js/ingest.js]: I believe that we can make this snippet clearer by adding some verbosity, e.g.

`approve_normalization_mscl = 'de909a42-c5b5-46e1-9985-c031b50e9d30'`

Or a comment highlighting the `de909a4` is the UUID for approve normalization from the MS chainlink table. 

Second, is there a reason for the construct to be:

`if 'constant' == 'value returned from function'` - I would normally expect it to be written:

`if (this.model.get('link_id') == 'value_we_want_to_confirm') `

(This seems to be the same for most of the calls below so this comment applies to those as well. My concern is that UUIDs are in a lot of places in Archivematica. If we can be as explicit as possible about the source of a UUID to future readers or folks new to the code (even if it is obvious to us), then it saves a little time for them.)

File: src/dashboard/src/media/js/ingest.js
Comment (ross-spencer) [inline src/dashboard/src/media/js/ingest.js]: Related to the above. This UUID is a chain not a chain-link, so it might be useful to elaborate on that.

**Review by ross-spencer — CHANGES_REQUESTED** [review]
Some changes requested to improve the documentation of the code now we're using UUIDs more widely but otherwise LGTM.

File: src/dashboard/src/components/transfer/views.py
Comment (sevein) [reply src/dashboard/src/components/transfer/views.py:159]: Yes, they're almost identical.

Comment (sevein) [reply src/dashboard/src/components/transfer/views.py:159]: During a big refactor that was never merged because of its size I tried to fix that:

https://github.com/artefactual/archivematica/pull/576/files#diff-254823927ae6fbe4584582f13c9b0c18R327

I learned the lesson and now I keep my work laser focused, e.g. see the second take of workflow translations here: https://github.com/artefactual/archivematica/issues/1101.

**Comment by sevein** [comment]
@ross-spencer ok noted and fixed, thanks. Ready to be reviewed again.

**Review by ross-spencer — APPROVED** [review]
:+1:

**Pull request merged**
